sys/linux/init.go

```diff
@@ -125,7 +124,6 @@ type arch struct {
 	FITHAW                      uint64
 	SNAPSHOT_FREEZE             uint64
 	SNAPSHOT_POWER_OFF          uint64
-	EXT4_IOC_SHUTDOWN           uint64
 	EXT4_IOC_RESIZE_FS          uint64
 	EXT4_IOC_MIGRATE            uint64
```

Comment:
And this one may not be needed anymore

Reply:
Done

Reply:
Oh, wait, that's a different one. Let me send another patch.